book/UpperLimitsTable.ipynb

```diff
@@ -674,10 +525,10 @@
     "table.add_column(\"Signal Region\", justify=\"left\", style=\"cyan\", no_wrap=True)\n",
     "table.add_column(\"Total Bkg.\", justify=\"center\")\n",
     "table.add_column(\"Data\")\n",
-    "table.add_column(\"XSec\")\n",
-    "table.add_column(\"S95Obs\")\n",
-    "table.add_column(\"S95Exp\")\n",
-    "table.add_column(\"CLb(S95Obs)\")\n",
+    "table.add_column(r\"XSec \\[fb]\")\n",
+    "table.add_column(\"S 95 Obs\")\n",
+    "table.add_column(\"S 95 Exp\")\n",
+    "table.add_column(\"CLb\")\n",
```

Comment:
this change has to be reverted. There's nothing that explicitly mentions that the CLb is evaluated at the observed upper limit versus the expected upper limit, and not having it in the table doesn't make it apparent.

Reply:
Reverted to `CLb(S 95 Obs)`

Reply:
This is one of the frustrations with the current description in the various tables across various papers -- it is not explicit or specific enough -- and that's what motivated this notebook.